src/Analyzers/Core/Analyzers/SimplifyInterpolation/Helpers.cs

```diff
@@ -14,9 +14,14 @@
 
 namespace Microsoft.CodeAnalysis.SimplifyInterpolation
 {
-    internal static class Helpers
+    internal class Helpers
     {
-        public static void UnwrapInterpolation<TInterpolationSyntax, TExpressionSyntax>(
+        protected virtual SyntaxNode GetPreservedInvocationInstanceSyntax(IInvocationOperation invocationOperation)
+        {
+            return invocationOperation.Instance.Syntax;
+        }
```

Comment:
far prefer this to be abstract.  make bth subclasses be explicit about the behavior they want.

Reply:
(Inheritance was removed.)